app/components/post_list/post_list.tsx

```diff
@@ -65,8 +65,8 @@ type ScrollIndexFailed = {
 };
 
 const CONTENT_OFFSET_THRESHOLD = 160;
+const SCROLL_EVENT_THROTTLE = Platform.select({android: 17, default: 60});
```

Comment:
Why not 16 which is the default? And how does this affect the rest of the scroll functionality on Android? Specially the frame drops while scrolling?

Reply:
I thought default is 0.  The note says Values <= 16 will disable throttling which is what I'm trying to avoid.

Plus I also made sure that the setStates are not being queued even though they won't change values as much as I can (adding if logics).

https://reactnative.dev/docs/scrollview#scrolleventthrottle